src/index.tsx

```diff
@@ -183,29 +186,41 @@ class Cropper extends React.Component<CropperProps, State> {
   }
 
   onMediaLoad = () => {
-    this.computeSizes()
+    const cropSize = this.computeSizes()
     this.emitCropData()
-    this.setInitialCrop()
+    this.setInitialCrop(cropSize as Size)
```

Comment:
I don't like to use `as`. Here, in the unlikely case where `cropSize` would be `undefined`, I would prefer to not call `this.setInitialCrop()`.

Reply:
Yeah, you're right. Updated